src/app/main.cpp

```
@@ -215,6 +215,18 @@ int main(int argc, char *argv[])
     QCoreApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
 #endif
 
+    // Handle --version before creating QApplication to avoid
+    // initializing the platform/graphics stack just to print a version string.
+    {
+        QCoreApplication coreApp(argc, argv);
```

Clang (14.0.6)

[warning] variable 'coreApp' is not initialized

(cppcoreguidelines-init-variables)